spec/support/check.rb

```diff
@@ -8,23 +8,23 @@ def self.included(by)
 
     by.let(:shell) { FakeShell.new }
     by.let(:context) { Hashie::Mash.new }
-    by.subject { described_class.new(shell: shell, context: context) }
+    by.let(:check) { described_class.new(shell: shell, context: context) }
 
-    by.let(:perform) { silence_stream(STDOUT) { subject.call } }
+    by.subject(:perform) { silence_stream(STDOUT) { check.call } }
   end
 
   module Dsl
     def it_expects_check_to_fail(message = nil)
-      it { expect { perform rescue nil }.not_to change { context } }
+      it { expect { subject rescue nil }.not_to change { context } }
```

Comment:
This allows us to use the `it_expects_check_to_fail` helper even when within a different `subject`.